ui/src/app/kvm/components/KVMActionFormWrapper/ComposeForm/InterfacesTable/SubnetSelect/SubnetSelect.test.tsx

```diff
@@ -172,6 +175,66 @@ describe("SubnetSelect", () => {
       .find("SubnetSelect ContextualMenu")
       .prop("links") as MenuLink[];
     filteredLinks = links.filter((link) => typeof link !== "string");
-    expect(filteredLinks.length).toBe(2);
+    expect(filteredLinks.length).toBe(1);
+  });
+
+  it.only("shows an error if multiple interfaces defined without at least one PXE network", async () => {
+    const fabric = fabricFactory();
+    const space = spaceFactory();
+    const [pxeVlan, nonPxeVlan] = [
+      vlanFactory({ fabric: fabric.id, id: 1 }),
+      vlanFactory({ fabric: fabric.id, id: 2 }),
+    ];
+    const [pxeSubnet, nonPxeSubnet] = [
+      subnetFactory({ name: "pxe", vlan: pxeVlan.id }),
+      subnetFactory({ name: "non-pxe", vlan: nonPxeVlan.id }),
+    ];
```

Comment:
Curious as to the benefit of defining these in arrays but only assigning them to decomposed variables.

Reply:
Not much benefit I suppose, just seems nice to define both items in one declaration